components/TextFields/examples/TextFieldKitchenSinkExample.swift

```diff
@@ -96,16 +97,37 @@ final class TextFieldKitchenSinkSwiftExample: UIViewController {
     setupExampleViews()
   }
 
-  func setupDefaultTextFields() -> [MDCTextInputControllerLegacyDefault] {
+  func setupBoxTextFields() -> [MDCTextInputControllerTextFieldBox] {
+    let textFieldBox = MDCTextField()
+    scrollView.addSubview(textFieldBox)
+    textFieldBox.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
When we addSubview a setNeedsLayout is implicitly invoked. 
Does the state of .translatesAutoresizingMaskIntoConstraints matter at this point or does it not matter until the layout() actually happens?  To put it another way, will constraints get created, then disabled after the .translatesAutoresizingMaskIntoConstraints is set to false?
I typically set .translatesAutoresizingMaskIntoConstraints = false before addSubview just in case, but I have no evidence that I should be.  Curious if you knew either way.

Reply:
I have never checked! Would be really good to know.

Reply:
I poked around and this use does not create constraints yet. However, I think it makes sense. What if a layout pass happened when we didn't want it yet? Would be good best practice to make sure you turn tamic on or off before adding as a subview.